Assets/SequenceSDK/Authentication/ILogin.cs

```diff
@@ -68,6 +68,10 @@ public interface ILogin
         /// </summary>
         public void AppleLogin();
 
+        /// <summary>
+        /// Returns true iff there is an authentication attempt in process
```

Comment:
iff > if

Reply:
I was thinking iff as in the short form for "if and only if". I guess that's not as common nomenclature as I thought it was. I'll change it thanks

Reply:
Ah I hadn't heard about it before, TIL!